### PR TITLE
Fill in setting information

### PR DESCRIPTION
**Pull request by danhp**
The page is no longer blank. But some things like themes still are as they aren't completed yet.

### CODE REVIEW

File: src/client/settings/_settings.coffee [hunk 1]
```diff
@@ -26,10 +26,17 @@ angular.module 'app.settings', []
   vm.load = ->
     $http.get('/containers/settings/settings').success (data) ->
       vm.settings = data
- 
+
+  vm.start_screens = ["Movies", "TV Series", "Anime"]
+
+  vm.movies_quality =
+    all: 'All'
```
Comment (hadim) [inline src/client/settings/_settings.coffee]: make All a string no ?

File: src/client/settings/_settings.coffee [hunk 2]
```diff
@@ -26,10 +26,17 @@ angular.module 'app.settings', []
   vm.load = ->
     $http.get('/containers/settings/settings').success (data) ->
       vm.settings = data
- 
+
+  vm.start_screens = ["Movies", "TV Series", "Anime"]
```
Comment (hadim) [inline src/client/settings/_settings.coffee:30]: for now we can let like that but in the future it should based on plugin available i guess.

**Comment by hadim** [comment]
Thank you !

File: src/client/settings/_settings.coffee (hunk 1 above)
Comment (danhp) [inline src/client/settings/_settings.coffee]: sure.

File: src/client/settings/_settings.coffee [hunk 3]
```diff
@@ -26,10 +26,17 @@ angular.module 'app.settings', []
...
+
+  vm.movies_quality =
+    all: "All"
```
Comment (hadim) [inline src/client/settings/_settings.coffee]: I mean why not "all": "All" ?

**Comment by phamdaniel** [comment]
Think that covers it for now.

File: src/client/settings/_settings.coffee (hunk 3 above)
Comment (danhp) [inline src/client/settings/_settings.coffee]: aha
I can do that for consistency.The other two keys needed it because of how coffee-script parsed them

**Comment by danhp** [comment]
Hope this get it :smile:

**Pull request merged**